project.clj

```diff
@@ -9,10 +9,10 @@
    :password :env/nexus_jenkins_password
    :sign-releases false})
 
-(def tk-version "1.1.1")
-(def tk-jetty9-version "1.3.1")
-(def ks-version "1.1.0")
-(def tk-status-version "0.2.1")
+(def tk-version "1.2.0")
```

Comment:
Not only that, there are some dependency complexities when I do this that may or may not take time to resolve, I say lets hold off on it. This patch was about plucking the majority of the low hanging fruit, I think we can dive into the individual troublesome pieces that are left as separate work.